Gemfile

```diff
@@ -34,7 +34,7 @@ else
   gem 'gds-sso', '13.2.0'
 end
 
-gem 'airbrake', '~> 4.3.4'
+gem 'airbrake', github: 'alphagov/airbrake', branch: 'silence-dep-warnings-for-rails-5'
```

Comment:
Do you know if the intention is for this branch to stay around for a while? I wonder if we could wrap its silencing behaviour in something that checks if the Rails version is 5 and then merge it into master?

This is probably out of scope for this upgrade, though.